tools/clang/unittests/HLSLExec/ShaderOpArith.xml

```diff
@@ -2487,6 +2487,17 @@
       </RootValues>
       <Shader Name="CS" Target="cs_6_6">
           <![CDATA[// Shader source code will be set at runtime]]>
+      </Shader>      
+  </ShaderOp>>
+  
+  <ShaderOp Name="WaveSizeRangeTest" CS="CS">
```

Comment:
There doesn't appear to be any reason to have this copy beyond the WaveSizeTest template here.  You shouldn't need an independent entry unless there are independent configuration requirements defined in this file.

Reply:
The extra configuration requirement is setting Target="cs_6_8"
I need to make it separate because of the nature of how the source is generated (wavesize parameters being defined via compile define flags).